mopro-ios/MoproKit/Example/MoproKit/ViewController.swift

```diff
@@ -78,15 +78,15 @@ class ViewController: UIViewController {
         navigationController?.pushViewController(keccakZkeyVC, animated: true)
     }
 
-    @objc func openRSA() {
-        let rsaVC = RSAViewController()
-        navigationController?.pushViewController(rsaVC, animated: true)
-    }
+    // @objc func openRSA() {
```

Comment:
Do we need to disable this and anon-aadhaar in example app? It is just a view, and useful for people who want to experiment with it. Shouldn't impact CI test time I think?

Reply:
Just enable this function but not tests for it?

Reply:
Yes, when you build the app it is useful. Example app is useful for a lot more than just CI.

Reply:
reverted